src/main/frontend/db/query_react.cljs

```diff
@@ -16,40 +16,56 @@
             [lambdaisland.glogi :as log]))
 
 (defn resolve-input
-  [input]
-  (cond
-    (= :right-now-ms input) (util/time-ms)
-    (= :start-of-today-ms input) (util/today-at-local-ms 0 0 0 0)
-    (= :end-of-today-ms input) (util/today-at-local-ms 24 0 0 0)
+  ([input]
+   (resolve-input input nil))
+  ([input current-block-uuid]
+   (cond
+     (= :right-now-ms input) (util/time-ms)
+     (= :start-of-today-ms input) (util/date-at-local-ms 0 0 0 0)
+     (= :end-of-today-ms input) (util/date-at-local-ms 24 0 0 0)
 
-    (= :today input)
-    (date->int (t/today))
-    (= :yesterday input)
-    (date->int (t/minus (t/today) (t/days 1)))
-    (= :tomorrow input)
-    (date->int (t/plus (t/today) (t/days 1)))
-    (= :current-page input)
-    (some-> (or (state/get-current-page)
-                (:page (state/get-default-home))
-                (date/today)) string/lower-case)
+     (= :today input)
+     (date->int (t/today))
+     (= :yesterday input)
+     (date->int (t/minus (t/today) (t/days 1)))
+     (= :tomorrow input)
+     (date->int (t/plus (t/today) (t/days 1)))
+     (= :current-page input)
+     (some-> (or (state/get-current-page)
+                 (:page (state/get-default-home))
+                 (date/today)) string/lower-case)
+     (= :current-block input)
```

Comment:
The logseq.api also uses `resolve-input`. Since `current-block-uuid` is nil for it, these next two `cond` conditions would fail and possibly confuse plugin authors. Could you update them to also check for `current-block-uuid`?